src/site/twitch.tv/modules/chat/ChatController.vue

```diff
@@ -101,6 +102,11 @@ const ignoreClearChat = useConfig<boolean>("chat.ignore_clear_chat");
 // Defines the current channel for hooking
 const currentChannel = ref<CurrentChannel | null>(null);
 
+// Community tab handler
+const communityTabToggleButton = document.querySelector("[data-test-selector='chat-viewer-list']");
```

Comment:
Please avoid DOM operations. We've built an array of tools which are capable of directly hooking into Twitch's react instance. See `src/common/ReactHooks.ts` & `src/common/Reflection.ts`.

Furthermore, it's possible that components already hooked here may hold the method to this specific button.

Reply:
@AnatoleAM Rewrite.